### PR TITLE
tests/gcoap_fileserver: only enable test with GCC

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description
We see a lot of failures on this on CI with LLVM.
Blacklist the LLVM build until someone manages to find out what the problem is.

### CODE REVIEW

File: tests/net/gcoap_fileserver/Makefile [hunk 1]
```diff
@@ -23,6 +23,8 @@ USEMODULE += shell_cmd_md5sum
 
 # automated test only works on native
 TEST_ON_CI_WHITELIST += native
+# FIXME: for some reson the test fails very often when build with clang
```
Comment (aabadie) [inline tests/net/gcoap_fileserver/Makefile:26]:
```suggestion
# FIXME: for some reson the test fails very often when built with clang
```

File: tests/net/gcoap_fileserver/Makefile [hunk 2]
```diff
@@ -23,6 +23,8 @@ USEMODULE += shell_cmd_md5sum
 
 # automated test only works on native
 TEST_ON_CI_WHITELIST += native
+# FIXME: for some reson the test fails very often when built with clang
```
Comment (aabadie) [inline tests/net/gcoap_fileserver/Makefile:26]:
```suggestion
# FIXME: for some reason the test fails very often when built with clang
```

Sorry, just spotted this one...

**Review by aabadie — APPROVED** [review]

**Comment by aabadie** [comment]
bors merge

**Comment by benpicco** [comment]
bors merge

**Pull request merged**